python/CMakeLists.txt

```diff
@@ -406,7 +406,7 @@ foreach(pyfile ${PY_FILES})
     COMMAND ${CMAKE_COMMAND} -E make_directory "${QGIS_PYTHON_OUTPUT_DIRECTORY}"
     COMMAND ${CMAKE_COMMAND} -E copy ${pyfile} "${QGIS_PYTHON_OUTPUT_DIRECTORY}"
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-    DEPENDS ${pyfile}
+    # DEPENDS ${pyfile}
```

Comment:
What's the story here? Does newer cmake auto-determine this?

Reply:
no, that has never been valid and is ignored: cf https://cmake.org/cmake/help/latest/command/add_custom_command.html#build-events
We'd need using some other CMake syntax to handle dependencies

Reply:
Looks like it was there up till 3.31: https://cmake.org/cmake/help/v3.31/command/add_custom_command.html#depends

Reply:
Ok, thanks for the explanation! If this was never working then let's just remove those lines instead of commenting?

Reply:
yes I had an hesitation if we'd want to keep them as a memory that we may want to do something more involved to track dependencies. But I'm fine removing them too